compact/pfrich.xml

```diff
@@ -151,26 +151,24 @@
 ##### Sensor plane
 - sensors will be placed on a plane
   - plane dimensions:
-    - `sensordist`: distance between sensor plane active surface (e.g., photocathode) and aerogel backplane
     - `rmin`: minimum radial position of a sensor's centroid
     - `rmax`: maximum radial position of a sensor's centroid
 </documentation>
 <plane
-  sensordist="PFRICH_sensor_dist"
   rmin="PFRICH_rmin1 + 2*cm"
   rmax="PFRICH_rmax  - 4*cm"
   />
 
 <services>
   <comment>
-    Material should be equivallent with 3x0.5cm Al, spread over the entire available distance.
+    Material should be equivallent with 3x0.5cm Al, spread over the entire available distance (`PFRICH_services_length`).
     This was confirmed by Silvia.
   </comment>
-  <component name="aluminum" thickness="5*mm" vis="RICHServiceVis" material="Aluminum"/>
-  <component name="air" thickness="40*mm" vis="InvisibleNoDaughters" material="Air"/>
-  <component name="aluminum" thickness="5*mm" vis="RICHServiceVis" material="Aluminum"/>
-  <component name="air" thickness="40*mm" vis="InvisibleNoDaughters" material="Air"/>
-  <component name="aluminum" thickness="5*mm" vis="RICHServiceVis" material="Aluminum"/>
+  <component name="aluminum" thickness="0.5*cm" vis="RICHServiceVis" material="Aluminum"/>
+  <component name="air" thickness="(PFRICH_services_length - 3*0.5*cm) / 2.0" vis="InvisibleNoDaughters" material="Air"/>
+  <component name="aluminum" thickness="0.5*cm" vis="RICHServiceVis" material="Aluminum"/>
+  <component name="air" thickness="(PFRICH_services_length - 3*0.5*cm) / 2.0" vis="InvisibleNoDaughters" material="Air"/>
```

Comment:
Is this correct? I assume 3*0.5*cm is to account for the three aluminum layers. But isn't 0.5*cm the half-length, so shouldn't this be `(PFRICH_services_length - 3*1.0*cm) / 2.0`?

Reply:
0.5 cm is the full length; thicknesses are divided by 2 in `PFRICH_geo.cpp`

Reply:
Checking the numbers from the `placeVolume` calls, looks okay:
```
TEST                   service   Aluminum, thickness=0.500000, zpos=-7.250000
TEST                   service        Air, thickness=6.750000, zpos=-3.625000
TEST                   service   Aluminum, thickness=0.500000, zpos=0.000000
TEST                   service        Air, thickness=6.750000, zpos=3.625000
TEST                   service   Aluminum, thickness=0.500000, zpos=7.250000
TEST                   thickness_sum=7.500000, total_thickness=15.000000, global_zpos=-18.600000
```